### ISSUE TITLE
 Error in the add button image

### ISSUE THREAD

**Issue by magiaslab** [elegantthemes/Divi-Beta#235]
### Problem:

The text in the button add picture is wrong in the version in Italian

### Steps To Reproduce:
1.  
2. 
3. 
4. 

### Additional Information:

*Please include any of the following details that are relevant to the issue you are reporting.*

**Device**:

**Operating System**:

**Browser**:

**Screen Resolution**:

**Screenshot or Screencast**:
![schermata 2016-08-30 alle 09 53 54](https://cloud.githubusercontent.com/assets/9402517/18081028/485d73d4-6e98-11e6-81e0-0911698aef4e.png)

**Link To Download Layout JSON File**:

-

#### Before submitting your issue, please ensure the following:

* You searched this Github repo for the issue and found no open reports of the issue.
* You properly formated the issue title - eg. `Scope :: One sentence to describe the issue`.
* You have provided a detailed description of the issue.
* You have provided any/all relevant information: screenshot, screencast, page URL.
* You have provided the steps that one must take to reproduce the issue.

[View Example](https://github.com/elegantthemes/Divi-Beta/tree/master/.github/ISSUE_EXAMPLE.md)

**Comment by timayhelen** [comment]
@fikrirasyid Looks like translations inside the WordPress Media Upload modal are getting double encoded?

**Comment by fikrirasyid** [comment]
@timayhelen I'm currently testing encoding issue on all modules at this moment (I noticed it while working on https://github.com/elegantthemes/Divi-Beta/issues/236). I'll check the encoding on media uploader as well. I assume the image above appears on media uploader when editing gallery?

**Comment by fikrirasyid** [comment]
@timayhelen I've submitted https://github.com/elegantthemes/submodule-builder/pull/1036 for this. It wasn't double encoded, but the translated string hasn't been decoded before being displayed.

@magiaslab We've submitted a patch for this and it is currently under review process. Once the patch passes review process, it should be merged on the next beta release. Thank you for reporting the issue 😄
